examples/docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -0,0 +1,113 @@
+---
+import { SIDEBAR } from '../../config.ts';
```

Comment:
looks like you made some changes in this file, but lets keep the components unchanged. Our eventual goal is to turn this example into a collection of components that our docs site can import, and that way we won't need to maintain the same site twice. 

If you made changes to other components, try to remove those as well and just use what's in `docs/src/components`. If anything doesn't work / changes are needed, call them out and we can try to make a change in both places.

Reply:
I had to change the Sidebar because I needed a `index.astro` page in order to support multiple languages, I followed the conventions setup in the `docs/` folder, but it might be better to tell users to setup redirect rules in the server, and if possible to setup redirect rules in Astro's dev server.

Reply:
@FredKSchott How should I explain `Internationalization` to users using the docs site, and also, I don't really know French or Spanish, would it be possible to get other devs who know those languages to convert the `example/docs` to those languages?

Reply:
It could be nice to have the sample content in the docs example translated :)

Reply:
Yeah

Reply:
you could do it with google translate, or just do some `lorem ipsum`

Reply:
> I had to change the Sidebar because I needed a index.astro page in order to support multiple languages.

Check out `src/pages/index.astro` in the main `docs/` site. We actually have an automatic redirect from the homepage, that then redirects you to `/{YOUR-LANGUAGE}/getting-started`. I think that would be a great default for the docs template, too.

Reply:
I created a slightly unique file structure for the `examples/docs` folder, for this to work properly. Check out my latest changes.

Reply:
+1! I actually really like this organization and eventually want to make the same change to docs.astro.build (move english docs to -> `/en/`).